### PR TITLE
feat: add icons for new bottom bar

### PR DESCRIPTION
**Pull request by jimut-dialpad**
## Description
Reason: https://dialpad.atlassian.net/browse/UC-13921
Design: 
https://www.figma.com/file/PTGerqq0CvhbWQSXNPoMAV/%5BUC-13921%5D-Bottom-Bar-Redesign
https://www.figma.com/file/6JnU4TNxYXIrjqH381nKdy/%5BUC-13402%5D-Raise-Hand-Flow
Icon Library: https://www.figma.com/file/FHMdVqC8qWcEn9Ox1s6k2e/Icons?node-id=0%3A1

## Pull Request Checklist

 - [ ] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: docs/_data/svg-system.json [hunk 1]
```diff
@@ -515,6 +521,18 @@
...
     "desc": "Communicates the user can add users to a group of people."
   },
+  {
+    "name": "Make Co-Host",
+    "file": "make-co-host",
+    "vue": "IconMakeCoHost",
+    "desc": "Communicates the user can make another participant a co-host."
+  },
+  {
+    "name": "Remove Co-Host",
```
Comment (josedialpad) [inline docs/_data/svg-system.json]: Can we maintain the alphabetical order on these? If we want to keep them grouped, consider using `Co-Host, Make` and `Co-Host, Remove` as name.

**Review by josedialpad — COMMENTED** [review]
One comment, also the PR should meet the [commit message convention](https://github.com/dialpad/dialtone/blob/staging/.github/COMMIT_CONVENTION.md#git-commit-message-convention). Can you please update the PR title and the commit message?

File: docs/_data/svg-system.json (hunk 1 above)
Comment (jimut-dialpad) [reply docs/_data/svg-system.json]: Updated accordingly.

**Comment by jimut-dialpad** [comment]
Updated the PR title. Is this okay?
Added one more icon that was left out.

**Comment by josedialpad** [comment]
> Updated the PR title. Is this okay? Added one more icon that was left out.

Yeah looks good, thanks

**Pull request merged**